pipenv/progress.py

```diff
@@ -13,11 +13,12 @@
 import sys
 import time
 import crayons
+from .environments import PIPENV_COLORBLIND
 
 STREAM = sys.stderr
 MILL_TEMPLATE = '%s %s %i/%i\r'
 DOTS_CHAR = '.'
-if os.name != 'nt':
+if os.name != 'nt' and not PIPENV_COLORBLIND:
     BAR_FILLED_CHAR = str(crayons.green('▉', bold=True))
     BAR_EMPTY_CHAR = str(crayons.black('▉'))
 else:
```

Comment:
Probably need to extend the if-else branching:

* if Windows: use ASCII characters.
* else if colorblind: use Unicode blocks without colors.
* else: call crayons.

Reply:
Ok, evolving that a bit:
 - if Windows: use ASCII characters.
 - else:
    - if hide_emojis: use ASCII characters
    - else: use Unicode blocks
    - if not colorblind: call crayons.

Yes?